Wordle.UI/Menu.cs

```diff
@@ -0,0 +1,105 @@
+namespace Wordle.UI;
+
+public class Menu
+{
+    public bool WelcomeMenu()
+    {
+        Console.ForegroundColor = ConsoleColor.White;
+        Console.BackgroundColor = ConsoleColor.Black;
+        Console.Clear();
+        Logo(Console.ForegroundColor, Console.BackgroundColor);
+        Console.WriteLine(@"
+Choose an option from the following list and press enter:
+    [ -g | --game ] to play the game
+    [ -h | --help ] for usage/instructions
+    [ -q | --quit ] to quit
+What would you like to do? ");
+        switch (Console.ReadLine())
+        {
+            case "-g":
+            case "--game":
+                Play play = new Play();
+                play.NewWordleGame();
+                return false;
+            case "-h":
+            case "--help":
+                HelpMenu();
+                return true;
+            case "-q":
+            case "--quit":
+                return false;
+            default:
+                return true;
+        }
+    }
+    private void Logo(ConsoleColor foreground, ConsoleColor background)
+    {
+        ConsoleColor[] colors = new ConsoleColor[] { ConsoleColor.DarkGreen, ConsoleColor.DarkGray, ConsoleColor.DarkYellow };
+        Console.WriteLine("Welcome to" + Environment.NewLine);
+        Console.WriteLine("╔═══╦═══╦═══╦═══╦═══╦═══╗");
+        foreach (char letter in "WORDLE")
+        {
+            Console.Write("║");
+            Random random = new Random();
```

Comment:
This is cool! I tried a few incorrect menu options and it changed the Logo at the top - didn't expect that!